src/hooks/useTimeToLive.ts

```diff
@@ -7,10 +7,13 @@ import { combineContractReads } from '../lib/wagmi'
 import { useLastPrice } from './useLastPrice'
 import { useRemainingBalance } from './useRemainingBalance'
 
-export const useTimeToLive = (batchId: string, watch = true) => {
+export const useTimeToLive = (
+	batchId: string,
+	amount: number,
```

Comment:
The `amount` argument isn't used in the hooks either, so this file should probably be reverted. Except if you want to move the logic to the `useTimeToLive` hook instead of the `TimeToLive` component.